cli/azd/pkg/project/framework_service_maven.go

```diff
@@ -153,6 +174,58 @@ func (m *mavenProject) Package(
 	}, nil
 }
 
+// funcAppDir returns the directory of the function app packaged by azure-functions-maven-plugin for the given service.
+//
+// The app is typically packaged under target/azure-functions.
+func (m *mavenProject) funcAppDir(ctx context.Context, svc *ServiceConfig) (string, error) {
```

Comment:
can you add unit test for this function?

Reply:
added some basic scenario coverage for funcapp in framework_service_maven_test.go.